public/js/login.js

```diff
@@ -1,25 +1,35 @@
 $(document).ready(function () {
     $("#login_button").on("click", function () {
-        var username = $("#username").val()  //Change the ids here if you choose 
-        var password = $("#password").val()  //to change the views later.
-        login(username, password) //Logs in the user.
+        login() //Logs in the user.
+    })
+
+    $("#username, #password").on("change", function () {
+        $(document).keypress(function (e) {
+            if (e.which == 13) {
+                login()
+            }
+        })
     })
 })
 
 function login(username, password) {
```

Comment:
Why does this function have parameters?
```suggestion
function login() {
```